frontend/src/app/modules/availability/new-slot/general/general.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-general',
+    templateUrl: './general.component.html',
+    styleUrls: ['./general.component.sass'],
+})
+export class GeneralComponent {
+    public slotSizes: string[] = ['30 min', '60 min'];
+
+    public selectedSlotSize: string = this.slotSizes[0];
+
+    public slotsFrequencies: string[] = ['30 min', '60 min'];
+
+    public selectedFrequency: string = this.slotsFrequencies[0];
+
+    public meetingLocations: string[] = ['Google Meet', 'Zoom'];
+
+    public selectedLocation: string = this.meetingLocations[0];
+
+    public meetingPaddings: string[] = ['15 min', '30 min'];
+
+    public selectedMeetingPadding: string = this.meetingPaddings[0];
+
+    public slotActivityOptions: string[] = ['Days', 'Range', 'Indefinitely'];
+
+    public selectedSlotActivity: string = this.slotActivityOptions[0];
+
+    public slotActivityValue: string;
+
+    public meetingName: string;
+
+    public maxBookings: number;
+
+    public minBookingMeetingDifferences: string[] = ['2 hours', '4 hours'];
+
+    public selectedBookingMeetingDifference: string = this.minBookingMeetingDifferences[0];
+
+    public hideFromCommon: boolean = false;
+
+    public addAdvanced: boolean = true;
```

Comment:
It would be better to create an interface for all fields that are related to general settings and use this interface here

Reply:
Shouldn't we wait until backend entities are ready before creating frontend interfaces for them?

Reply:
We can change an interface at any time, but with the interface, this component will look better

Reply:
Ok